docs/specs/bot-protection.md

```diff
@@ -122,9 +150,7 @@ Bot protection is supported only in the following step types:
 
 - `identify` in `signup`, `promote`, `login`, `signup_login`, and `account_recovery`.
 - `authenticate` in `login` and `reauth`.
-
-Note that the step type `create_authenticator` in `signup` and `promote` does not support bot protection.
-Bot protection should be configured at a previous step, for example `identify`.
+- `create_authenticator` in `signup` and `promote`
```

Comment:
@louischan-oursky do you think we support all below

1. `create_authenticator_oob_otp`
2. `create_authenticator_password`
3. `create_authenticator_totp`

Or just 1? 🙏

Reply:
According to line 219, just Point 1 is supported.